src/core/os_macos.c

```diff
@@ -53,11 +57,38 @@ void os_sleep(double seconds) {
 }
 
 void os_request_permission(os_permission permission) {
-  //
+  if (permission == OS_PERMISSION_AUDIO_CAPTURE) {
+    // If on old OS without permissions check, permission is implicitly granted
+    if(![AVCaptureDevice respondsToSelector:@selector(authorizationStatusForMediaType:)]) {
+      if(state.onPermissionEvent) state.onPermissionEvent(permission, true);
```

Comment:
Should this have a `return`?  Is it supposed to run the `switch` below?

Reply:
It should have a return. Thanks!